files/image_config/logrotate/logrotate.d/rsyslog

```diff
@@ -85,7 +85,15 @@
     endscript
     postrotate
         if [ $(echo $1 | grep -c "/var/log/swss/") -gt 0 ]; then
-            pgrep -x orchagent | xargs /bin/kill -HUP 2>/dev/null || true
+            # for multi asic platforms, there are multiple orchagents
+            # send the SIGHUP only to the orchagent the which needs log file rotation
+            if [ $(pgrep -x orchagent | wc -l) -gt 0 ]; then
```

Comment:
if somehow only one orchagent starts in the multi-asic platform, this logic could do wrong classification. why not use the num_asic variable?

Reply:
If there is only on orchagent, the else block will take care of sending SIGHUP to the orchagent.

I tried using the using the num_asic by sourcing the asic.conf file  source doesn't seem to work in this logrotate script.

Reply:
updated to use the asic.conf in the latest commit